Description: <blankon branding>
 BlankOn logo in gnome control center details.
 .
 gnome-control-center (1:3.26.2-1blankon1) uluwatu; urgency=medium
 .
   * New upstream release
   * Add blankon_branding.patch
Author: AnTo SamalonA <ansamsoftdev@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-11-09

--- gnome-control-center-3.26.2.orig/panels/background/bg-wallpapers-source.c
+++ gnome-control-center-3.26.2/panels/background/bg-wallpapers-source.c
@@ -114,7 +114,7 @@ load_default_bg (BgWallpapersSource *sel
   for (i = 0; system_data_dirs[i]; i++) {
     filename = g_build_filename (system_data_dirs[i],
 				 "gnome-background-properties",
-				 "adwaita.xml",
+				 "blankon-wallpapers.xml",
 				 NULL);
     if (cc_background_xml_load_xml (self->priv->xml, filename)) {
       g_free (filename);
--- gnome-control-center-3.26.2.orig/panels/info/cc-info-panel.c
+++ gnome-control-center-3.26.2/panels/info/cc-info-panel.c
@@ -169,7 +169,7 @@ static void
 info_panel_setup_overview (CcInfoPanel  *self)
 {
   GtkWidget  *widget;
-
+  gtk_widget_hide (WID ("version_label"));
   widget = WID ("info_vbox");
   gtk_container_add (GTK_CONTAINER (self), widget);
 }
--- gnome-control-center-3.26.2.orig/panels/info/info-overview.ui
+++ gnome-control-center-3.26.2/panels/info/info-overview.ui
@@ -18,7 +18,7 @@
           <object class="GtkImage" id="system_image">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="resource">/org/gnome/control-center/info/GnomeLogoVerticalMedium.svg</property>
+            <property name="resource">/org/gnome/control-center/info/BlankonLogoBlank.png</property>
           </object>
           <packing>
             <property name="expand">False</property>
--- gnome-control-center-3.26.2.orig/panels/info/info.gresource.xml
+++ gnome-control-center-3.26.2/panels/info/info.gresource.xml
@@ -5,6 +5,6 @@
     <file preprocess="xml-stripblanks">info-overview.ui</file>
     <file preprocess="xml-stripblanks">info-default-apps.ui</file>
     <file preprocess="xml-stripblanks">info-removable-media.ui</file>
-    <file>GnomeLogoVerticalMedium.svg</file>
+    <file>BlankonLogoBlank.png</file>
   </gresource>
 </gresources>
